docs/reference/secrets-reference.md

```diff
@@ -0,0 +1,42 @@
+---
+id: secrets-reference
+title: Secrets YAML Reference
+---
+Recyclarr supports secret value substitution within it's YAML configuration files. This keeps sensitive
+values out of configuration files and makes it easier to share configurations without having to
+perform manual redaction. Using secrets is optional and support was introduced in version `TBD`.
+
+The `secrets.yml` file is located in the following locations (by default) depending on your
+platform:
+
+| Platform | Location                                               |
+| -------- | ------------------------------------------------------ |
+| Windows  | `%APPDATA%\recyclarr\secrets.yml`                     |
+| Linux    | `~/.config/recyclarr/secrets.yml`                     |
+| MacOS    | `~/Library/Application Support/recyclarr/secrets.yml` |
+
+Secrets can be used in both the Recyclarr config and settings files. Only one secrets file is supported.
+
+This file is optional and does not need to exist in order for Recyclarr to work properly.
```

Comment:
I feel like the format for this should be a list, like so. Let me know what you think.

```
The following rules apply:

- Secrets can be used in both the Recyclarr config and settings files.
- Only one secrets file is supported.
- This file is optional and does not need to exist in order for Recyclarr to work properly.
```